contract/tracking.proto

```diff
@@ -85,13 +85,14 @@ enum Intent {
   INTENT_APPOINTMENT_BOOKING = 10;
   INTENT_FIND_HELP = 11;
   INTENT_APPOINTMENT_RESCHEDULE = 12;
-  INTENT_CREATE_HOME_INSURANCE_QUOTE = 13;
+  INTENT_CREATE_INSURANCE_QUOTE = 13;
```

Comment:
i think it's important to not change this so it's backwards compatible (messages already sent or services using the old version stauing compatible with the new ones). Could you add `INTENT_CREATE_INSURANCE_QUOTE` as number 20?

Reply:
It was previously 13, so this basically reverts it to what it was before. You'll see I've moved the `INTENT_CREATE_HOME_INSURANCE_QUOTE` to 20 where it should have been added previously

Reply:
See previous PR https://github.com/utilitywarehouse/customer-tracking-sdk/pull/127